docs/terms/mlms.md

```diff
@@ -0,0 +1,9 @@
+---
+id: mlms
+title: Masked Language Models (MLMs)
+hoverText: MLMs are a type of NLP model, which have a special token, which is replaced with a word from the vocabulary. The model then predicts the word that was masked.
+---
+
+### Term explanation
+
+MLMs are a type of NLP model, which have a special token, which is replaced with a word from the vocabulary. The model then predicts the word that was masked. 
```

Comment:
Is it possible to reduce two duped def here.

Reply:
No, those are two different things. 

One is the tooltip text, the other is the text (which can be as long as you want and have any formatting) which appears in the page if you click on the tooltip.

Reply:
(By the way, I created this term only as an example for showing how it works)